airflow/providers/google/suite/hooks/calendar.py

```diff
@@ -0,0 +1,213 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+"""This module contains a Google Calendar API hook"""
+
+from datetime import datetime
+from typing import Any, Dict, Optional, Sequence, Union
+
+from googleapiclient.discovery import build
+
+from airflow.exceptions import AirflowException
+from airflow.providers.google.common.hooks.base_google import GoogleBaseHook
+
+
+class GCalendarHook(GoogleBaseHook):
```

Comment:
Why not just `CalendarHook` ?

Reply:
I think `GoogleCalendarHook` would be most aligned with other hooks:
https://github.com/apache/airflow/blob/57ed3abe2d3c3455a635d87ecd1d2064449f75a2/airflow/providers/google/marketing_platform/hooks/campaign_manager.py#L28
https://github.com/apache/airflow/blob/57ed3abe2d3c3455a635d87ecd1d2064449f75a2/airflow/providers/google/marketing_platform/hooks/analytics.py#L26
https://github.com/apache/airflow/blob/57ed3abe2d3c3455a635d87ecd1d2064449f75a2/airflow/providers/google/marketing_platform/hooks/display_video.py#L27
https://github.com/apache/airflow/blob/57ed3abe2d3c3455a635d87ecd1d2064449f75a2/airflow/providers/google/suite/hooks/drive.py#L28

Reply:
Renamed to GoogleCalendar